src/bsp/src/stm32/hal/src/uart_print.c

```diff
@@ -2,7 +2,7 @@
 
 #include "hal/uart_print.h"
 
-#define CBUFF_HUART3_DATA_SIZE 128
+#define CBUFF_HUART3_DATA_SIZE 2048
```

Comment:
Was this for testing or you didn't have enough space?

Reply:
LWIP has assert prints that are way over 128 bytes. There is still a bug with them, but I think it doesn't hurt to have a little bigger buffer here